client/src/components/Page/ChooseYourRunPage.js

```diff
@@ -33,6 +34,7 @@ export default function ChooseYourRunPage() {
     event.preventDefault();
```

Comment:
If this is not a submit function for your form, I think you can remove it

Reply:
FYI 
Kept it in, so the page will not reload each time I submit the search.